pkg/apiserver/controllers/v1/decisions.go

```diff
@@ -54,7 +66,16 @@ func (c *Controller) GetDecision(gctx *gin.Context) {
 	} else {
 		PrometheusBouncersHasEmptyDecision(gctx)
 	}
-
+	go func() {
```

Comment:
I've move this call after the check on HEAD requests, because we don't send anything to the bouncer if the method is HEAD, I wouldn't really count it as a real pull (and I'm not sure bouncers will do HEAD requests by themselves ?)

Reply:
Fixed